package.json

```diff
@@ -40,6 +40,8 @@
         "test:e2e:clients": "npm run test:e2e:ganache; npm run test:e2e:geth:insta; npm run test:e2e:geth:auto",
         "test:e2e:chrome": "./scripts/e2e.chrome.sh",
         "test:e2e:firefox": "./scripts/e2e.firefox.sh",
+        "test:e2e:min": "./scripts/e2e.min.sh",
+        "test:e2e:surge": "./scripts/e2e.surge.sh",
```

Comment:
Detail but ``test:e2e:surge`` should probably be called ``test:e2e:cdn`` which is more descriptive.

Reply:
Changed to cdn in 2d43bb1489e4bf26bc8979d6db5cc37d5acce0f4